src/store/index.js

```diff
@@ -35,7 +30,7 @@ export default function () {
 
   const store = createStore(
     combineReducers(allReducers),
-    compose(enhancer, applyMiddleware(...middlewares))
+    composeWithDevTools(enhancer, applyMiddleware(...middlewares))
```

Comment:
Do you prefer dev tools to the logger? I liked the logger, but I'm not married to it (and the code to conditionally add it felt like unnecessary goo).

Reply:
I'm obsessed with redux devtools :) Remind me to show you why this Saturday

Reply:
Will do. I know you can time travel and also tap into them in prod sometimes :-)